src/customConfigTests/src/test/resources/config.properties

```diff
@@ -9,6 +9,7 @@ onetime_token.db.location=${onetime_token_test_db}
 token_expiry.minutes=${token_expiry_minutes}
 
 allow.localhost=${allow_localhost}
+proxy.dev.mode=${proxy_dev_mode}
```

Comment:
So we already have `allow.localhost`, can we just use that rather than adding a new config property?